src/pages/dar_collection_review/DarCollectionReview.js

```diff
@@ -76,41 +105,45 @@ export default function DarCollectionReview(props) {
     }
   }, [subcomponentLoading, isLoading]);
 
-  return (
-    div({style: Styles.PAGE}, [
-      div({}, [
-        h(ReviewHeader, {
-          darCode: collection.darCode || '- -',
-          projectTitle: darInfo.projectTitle || '- -',
-          downloadLink: h(ApplicationDownloadLink, {
-            darInfo,
-            researcherProfile,
-            datasets: collection.datasets || []
-          }),
-          isLoading,
-          redirectLink: h(RedirectLink, {user: currentUser, history: props.history})
+  return div({className: 'collection-review-page'}, [
+    div({className: 'review-page-header', style: { width: '90%', margin: '0 auto' }}, [
+      h(ReviewHeader, {
+        darCode: collection.darCode || '- -',
+        projectTitle: darInfo.projectTitle || '- -',
+        downloadLink: h(ApplicationDownloadLink, {
+          darInfo,
+          researcherProfile,
+          datasets: collection.datasets || [],
         }),
-        h(DataUseVoteSummary, {dataUseBuckets, isLoading}),
-        h(TabControl, {
-          labels: [tabs.applicationInformation, tabs.researchProposal],
-          selectedTab,
-          setSelectedTab,
-          isLoading
+        isLoading,
+        redirectLink: h(RedirectLink, {
+          user: currentUser,
+          history: props.history,
         }),
-        h(ApplicationInformation, {
-          isRendered: selectedTab === tabs.applicationInformation,
-          pi: darInfo.isThePi ? darInfo.researcher : darInfo.investigator,
-          institution: darInfo.institution,
-          researcher: darInfo.researcher,
-          email: darInfo.academicEmail,
-          piEmail: darInfo.piEmail,
-          city: `${darInfo.city}${!darInfo.state ? '' : ', ' + darInfo.state}`,
-          country: darInfo.country,
-          nonTechSummary: darInfo.nonTechRus,
-          department: darInfo.department,
-          isLoading: subcomponentLoading
-        })
-      ])
+      }),
+      h(DataUseVoteSummary, { dataUseBuckets, isLoading }),
+    ]),
+    div({ className: 'review-page-body', style: {padding: '1% 0% 0% 10%', backgroundColor: tabContainerColor} }, [ //TODO: take the margin measurements and apply as padding here
+      h(TabControl, {
+        labels: Object.values(tabs),
+        selectedTab,
+        setSelectedTab,
+        isLoading,
+        styleOverride: tabStyleOverride
+      }),
+      h(ApplicationInformation, {
+        isRendered: selectedTab === tabs.applicationInformation,
+        pi: darInfo.isThePi ? darInfo.researcher : darInfo.investigator,
+        institution: darInfo.institution,
+        researcher: darInfo.researcher,
+        email: darInfo.academicEmail,
+        piEmail: darInfo.piEmail,
+        city: `${darInfo.city}${!darInfo.state ? '' : ', ' + darInfo.state}`,
+        country: darInfo.country,
```

Comment:
`darInfo` in this case is the `DataAccessRequestData` object on `DataAccessRequest`. In that world, we're trying to deprecate the usages of city, state, country, etc in favor of getting that info directly from the researcher's `UserProperties`. We do have access to those in `researcherProfile` - it would be nice to get those values from there instead.